doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2646,6 +2648,51 @@ The fully expanded example above (without environment variables) looks like this
 Note: The ``id`` returned in the json response is the id of the file metadata version.
 
 
+.. _add-file-metadata-api:
+
+Adding File Metadata
```

Comment:
The documentation for "Adding file Metadata" was recently removed (in #9018) because it can't be used to add/change metadata for an existing file and is really only for registering files that were direct uploaded (and could now be used to register files after an out-of-band upload as well.) 

Can you perhaps look at modifying the [direct upload section](https://guides.dataverse.org/en/latest/developers/s3-direct-upload-api.html) to cover the out-of-band option rather than restoring the documentation here? Or some other alternative? Feel free to retitle that section to Direct/out-of-band DataFile Upload/Replace API, etc.

Also note that there are single file and multiple file variants of this call - I think only the multifile one was here in the native guide, but the direct upload section talks about both (and the replace file variants).

Reply:
I think it got re-added with a merge. I have removed it now.